index.js

```diff
@@ -1090,4 +1090,6 @@ exports.AggregateError = errorClasses.AggregateError;
 // Add all redis commands / node_redis api to the client
 require('./lib/individualCommands');
 require('./lib/extendedApi');
-require('./lib/commands');
+
+//enables adding new commands (for modules and new commands)
+exports.addCommand = require('./lib/commands')
```

Comment:
Currently the default is still to use underscores. But we support both, so please add a alias for it with underscore.